php/class-customize-snapshot-manager.php

```diff
@@ -1181,6 +1186,8 @@ function( $value ) {
 			wp_send_json_error( $data );
 		}
 
+		/** This filter is documented in wp-includes/class-wp-customize-manager.php */
+		$data = apply_filters( 'customize_save_response', $data, $this );
```

Comment:
Not quite, as `$this` is `CustomizeSnapshots\Customize_Snapshot_Manager` not `WP_Customize_Manager`. So you should replace `$this` with `$this->customize_manager`.